src/cmd/common.go

```diff
@@ -83,6 +83,13 @@ func deploy(bndlClient *bundle.Bundle) error {
 		// Determine the location of the extracted *.tf files
 		chdirFlag := fmt.Sprintf("-chdir=%s", filepath.Dir(bndlClient.GetDefaultExtractPath()))
 
+		// Configure tofu to use the terraformrc file that came with the bundle so it knows where to locate the providers
+		customRCPath := filepath.Join(filepath.Dir(bndlClient.GetDefaultExtractPath()), config.TerraformRC)
+		os.Setenv("TF_CLI_CONFIG_FILE", customRCPath)
```

Comment:
nit: we never unset these env vars. Could we build an input type for `udsEmbeddedTofu` with these values and then include them in the `osExec.Command` call by appending to the `cmd.Env`? 

Something like this (crude) example:

```
type tofuInput struct {
  Env map[string]string
}

input := tofuInput{}
input.Env["TF_CLI_CONFIG_FILE"] = customRCPath

useEmbeddedTofu(input)

// in useEmbeddedTofu

cmd := osExec.Command(// blah)
cmd.Env = os.Environ()
for key, val := range input.Env {
  cmd.Env[key]=val
}
cmd.Stdout = os.Stdout
cmd.Stderr = os.Stderr
err = cmd.Run()
return err

```

- https://pkg.go.dev/os/exec#example-Command-Environment